src/app/give-help/accommodation-form/accommodation-form.component.ts

```diff
@@ -22,13 +22,18 @@ export class AccommodationFormComponent {
     hostLanguage: [],
   });
   loading: boolean = false;
+  @ViewChild('phoneInput')
+  phoneInput!: { nativeElement: { value: any } };
```

Comment:
Dodaj proszę poprawny typ do `nativeElement` zamiast tworzyć swój. Cały DOM jest otypowany w TS.

Reply:
@mgol Jakby to miało wyglądać?

Reply:
To się zrobiło outdated, więc kontynuujmy w https://github.com/coi-gov-pl/gui-web-pomogamukrainie-forum/pull/135#discussion_r831989415.